content/integrations/postgresql.md

```diff
@@ -42,63 +40,87 @@ When it prompts for a password, enter the one used in the first command.
+        - relation_name: my_relation
           schemas:
-            - public
-            - prod
+            - my_schema_1
+            - my_schema_2
 
 * `collect_function_metrics` (Optional) - Collect metrics regarding PL/pgSQL functions from pg_stat_user_functions
-* `collect_count_metrics` (Optional) - Collect count metrics, default value is True for backward compatibility but they migth be slow, suggested value is False.
+* `collect_count_metrics` (Optional) - Collect count metrics. The default value is `True` for backward compatibility, but this might be slow. The recommended value is `False`.
+
+
+## Custom metrics
+
+Datadog will generate PostgreSQL metrics from custom query results. For each custom query, four components are required: `descriptors`, `metrics`, `query`, and `relation`.
+
+* `descriptors` is used to add tags to your custom metrics. It's a list of lists each containing 2 strings. The first string is for documentation purposes and usually used to make clear what we are getting from the query. The second string will be the tag name. Note that instance tags will automatically be added to that list.
+* `metrics` are key-value pairs where the key is the query column iname or column function and the value is a tuple containing the Datadog custom metric name and metric type (`RATE`, `GAUGE`, or `MONOTONIC`). In the example below, the results of the sum of the `idx_scan` column will appear in Datadog  with the metric name `postgresql.idx_scan_count_by_schema`.
+* `query` is where you'll construct the SQL query to obtain your custom metrics. Each column name in your SELECT query should have a corresponding item in the `descriptors` section. Each item listed in the Metrics section will be substituted for the first `%s` in the query.
+* `relation` indicates whether to include schema relations specified in the [`relations configuration option`](#configuration-options). If set to `true`, the second `%s` in the Query will be set to the list of schema names specified in the Relations configuration option.
+
+### Example 1
 
+    custom_metrics:
+      # All index scans & reads
+      - descriptors:
+          - [relname, schema]
+        metrics:
+            SUM(idx_scan) as idx_scan_count: [postgresql.idx_scan_count_by_schema, RATE]
+            SUM(idx_tup_read) as idx_read_count: [postgresql.idx_read_count_by_schema, RATE]
+        query: SELECT relname, %s FROM pg_stat_all_indexes GROUP BY relname;
+        relation: false
+
+The example above will run two queries in PostgreSQL:
+
+* `SELECT relname, SUM(idx_scan) as idx_scan_count FROM pg_stat_all_indexes GROUP BY relname;` which will generate the metric `postgresql.idx_scan_count_by_schema`.
+* `SELECT relname, SUM(idx_tup_read) as idx_read_count FROM pg_stat_all_indexes GROUP BY relname;` which will generate the metric `postgresql.idx_read_count_by_schema`.
 
-### Custom metrics
+Both metrics will use the tag `schema` with tag values from the results in the `relname` column.
```

Comment:
for example? e.g. `schema:<relname`

Since we allow value and value-less tags, it's best to spell out exactly what an example tags looks like. I recall a few support tickets where customers got confused about tag formats and found our documentation on it lacking.